packages/itwinui-css/src/icon/mixins.scss

```diff
@@ -3,50 +3,45 @@
 @import '../style/index';
 
 @mixin iui-icons-default {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-default;
   height: $iui-icons-default;
 }
 
 @mixin iui-icons-small {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-small;
   height: $iui-icons-small;
 }
 
 @mixin iui-icons-large {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-large;
   height: $iui-icons-large;
 }
 
 @mixin iui-icons-xl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-xl;
   height: $iui-icons-xl;
 }
 
 @mixin iui-icons-xxl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-xxl;
   height: $iui-icons-xxl;
 }
 
 @mixin iui-icons-3xl {
-  @include iui-icons-color();
-  display: inline-flex;
   width: $iui-icons-3xl;
   height: $iui-icons-3xl;
 }
 
+@mixin iui-icons-display {
```

Comment:
Is it bad if we include this to every icon? Because it feels that it should be used for every icon.

Reply:
yeah, Was thinking the same..
But then if parent of icon is not `display: flex`, `flex-shrink` rule would do nothing :/

Reply:
But it is not a big problem if it does nothing 😀

Reply:
This can't be used for every icon. It causes a lot of bad changes. Not every icon/element is currently using `display: inline-flex`, and adding this mixin to every icon breaks a lot of test images.